src/test/java/no/sikt/nva/DublinCoreParserTest.java

```diff
@@ -60,18 +63,41 @@ void shouldLogDcValuesThatAreNotUsedForScraping() {
             FIELD_WAS_NOT_SCRAPED_IN_LOCATION_LOG_MESSAGE, expectedDcValuedLogged, record.getOriginInformation())));
     }
 
+    @Test
+    void shouldConvertValidVersionToPublisherAuthority() {
+        var expectedPublisherAuthority = true;
+        var record = new Record();
+        record.setOrigin(Path.of("something/something"));
+        var dublinCore = DublinCoreFactory.createDublinCoreFromXml(new File(
+            VALID_DUBLIN_CORE), record.getOriginInformation());
```

Comment:
Istedet for å lese inn ei fil, generer et dublinCore object direkte med KUN de feltene som trengs.

Reply:
fixed